package.json

```diff
@@ -356,45 +356,67 @@
                     "description": "Run the linter on active files."
                 },
                 "julia.lint.missingrefs": {
-                    "type": "boolean",
-                    "default": true,
-                    "description": "Report possibly missing references."
+                    "type": "string",
+                    "default": "all",
+                    "enum": [
+                        "none",
+                        "nogetfield",
```

Comment:
Could we use something like `getfield missing`, or just generally a string that is less an identifier? I think this will show up in the UI, right? Or maybe there is a way to specify a display string for each of the values that can be chosen that would then appear in the UI?